Annotation/polar_golden_set.json

```diff
@@ -1728,25 +1910,38 @@
                     }
                 ],
                 "type": "polar",
-                "clause": "",
+                "clause": "whether she is indeed succumbed to his charm",
                 "embedded clauses": []
             }
         ]
     },
     "448": {
-        "sentence": "One study found that variations in the serotonin transporter gene influence whether or not a person will respond to an antidepressant such as an SSRI, which acts upon serotonin; variations in the norepinephrine transporter gene influence whether or not a person will respond to an antidepressant that acts mainly on the norepinephrine system, such as nortriptyline [ref].",

```

Comment:
Same comment as before - should we edit the sentences to be more grammatical but less realistic in a corpus context?